docs/clusterctl.md

```diff
@@ -0,0 +1,87 @@
+# clusterctl
+
+## Overview
+
+`clusterctl` simplifies getting started with cluster-api-provider-aws. It wraps
+the complicated steps into one command that will bring you from nothing to a
+fully bootstrapped Kubernetes cluster running on AWS.
```

Comment:
I think it would be good to expand this a bit to let the user know that the bootstrapped cluster is running the Cluster API components, and it is not just a "normal" k8s cluster.